lib/babe/babe.go

```diff
@@ -399,7 +399,7 @@ func (b *Service) initiate() {
 func (b *Service) initiateAndGetEpochHandler(epoch uint64) (*epochHandler, error) {
 	epochData, err := b.initiateEpoch(epoch)
 	if err != nil {
-		return nil, fmt.Errorf("failed to initiate epoch %d: %w", epoch, err)
```

Comment:
why remove epoch number?

Reply:
Because it's added in the `initiateEpoch` method, so to avoid repeating the epoch number a bunch of times in the error.

Anyway, the function handling the argument should be the only one adding context to the error with it, not its callers.